components/TriviaQuestions/index.js

```diff
@@ -1,40 +1,28 @@
-import React, { useState, useContext } from 'react';
```

Comment:
I moved much of the functional logic to the TriviaQ screen thinking it could be reusable in a sort of read-only mode  - could probably still use some improvements though.